traits/observers/_named_trait_observer.py

```diff
@@ -0,0 +1,52 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+from traits.observers._interfaces import IObserver
```

Comment:
On naming, should the module be named `_iobserver` and we have one module per interface object?

Reply:
I ended up naming this module to `_i_observer` (not the `_iobserver` I initially proposed), to be inlined with naming in pyface (maybe elsewhere too).